SEPADebit/src/main/java/com/braintreepayments/api/SEPADebitClient.java

```diff
@@ -47,7 +63,8 @@ public SEPADebitClient(@NonNull Fragment fragment, @NonNull BraintreeClient brai
      * @param listener a {@link SEPADebitListener}
      */
     public void setListener(SEPADebitListener listener) {
-
+        // TODO: handle delivering pending browser switch result when listener is set
```

Comment:
This TODO was to align with the other client `setListener` methods in #523 - but this case is a little different, since we will call `tokenize` after browser switching. In this case a merchant will have to set the listener to receive results from tokenize - does that seem like the right approach?